hybris/common/q/linker.cpp

```diff
@@ -1152,8 +1165,10 @@ static int open_library(android_namespace_t* ns,
   }
 
   // Otherwise we try LD_LIBRARY_PATH first, and fall back to the default library path
+  TRACE("[ opening %s from namespace %s from %s]", name, ns->get_name(),ns->get_ld_library_paths());
```

Comment:
These traces are commented out in later commit, should these be kept?

Reply:
@krnlyng, I bet it's safe to squash (=remove) those TRACE lines disabled by you.
This and two above in https://github.com/libhybris/libhybris/blob/637bb2dae8cf1c24a724ea8452ce02c06bec4979/hybris/common/q/linker.cpp#L1182

Reply:
krnlyng said it's best to keep these debug helpers, and not to comment them out. Fixed.